bedrock/firefox/templates/firefox/lockwise/lockwise.html

```diff
@@ -0,0 +1,105 @@
+            <div class="mzp-c-button-download-container">
+              <p >{{ _('Try Lockwise now') }}</p>
+              <a href="https://github.com/mozilla-lockwise/lockwise-addon/releases/download/2.2.5-alpha/lockwise-signed.xpi" class="mzp-c-button mzp-t-product" data-cta-type="Lockwise-Install" data-cta-position="Primary">{{ _('Install for Firefox') }}</a>
+            </div>
+          </div>
+          <div class="mzp-c-hero-cta add-on-download for-firefox-70-and-above" >
+            <div class="mzp-c-button-download-container">
+              <p >{{ _('Try Lockwise now') }}</p>
+              <a class="mzp-c-button mzp-t-product" id="lockwise-button" data-cta-type="Lockwise-Open" data-cta-position="Primary">{{ _('Open in Firefox') }} </a>
+            </div>
+          </div>
+          <div class="mzp-c-hero-cta add-on-download for-non-firefox-users" >
+            <div class="mzp-c-button-download-container">
+              <p >{{ _('Only in the Firefox browser') }}</p>
+              <a href="{{ url('firefox.download.thanks') }}" class="mzp-c-button mzp-t-product" data-cta-type="Lockwise-DownloadFX" data-cta-position="Primary">{{ _('Get Firefox') }}</a>
+            </div>
+          </div>
+          <div class="mzp-c-hero-image">
+          {% if LANG.startswith('en-') %}
+            <img src="{{ static('img/firefox/lockwise/EN.svg') }}" alt="">
+          {% elif LANG == 'de' %}
+            <img src="{{ static('img/fmake irefox/lockwise/DE.svg') }}" alt="">
+          {% elif LANG == 'fr' %}
+            <img src="{{ static('img/firefox/lockwise/FR.svg') }}" alt="">
+          {% elif LANG == 'it' %}
+            <img src="{{ static('img/firefox/lockwise/IT.svg') }}" alt="">
+          {% elif LANG.startswith('es-') %}
+            <img src="{{ static('img/firefox/lockwise/ES.svg') }}" alt="">
+          {% else %}
+            <img src="{{ static('img/firefox/lockwise/EN.svg') }}" alt="">
+          {% endif %}
+          </div>
+        </div>
+      </div>
+      <div class="mzp-c-hero-image">
+        <img src="" alt="">
+      </div>
+    </section>
+    <section class="dark-content">
+      <section class="mzp-l-card-third mzp-l-content">
+        <section class="mzp-c-card-picto ">
+          <div class="mzp-c-card-picto-content encryption">
+            <p class="picto-content">{{ _('256-bit encryption protects you while syncing') }}</p>
+          </div>
+        </section>
+        <section class="mzp-c-card-picto">
+          <div class="mzp-c-card-picto-content touch">
+            <p class="picto-content">{{ _('Get to your passwords securely with Face or Touch ID') }}</p>
+          </div>
+        </section>
+        <section class="mzp-c-card-picto">
+          <div class="mzp-c-card-picto-content privacy">
+            <p class="picto-content"><span>{{ _('Your privacy comes first. ')}}</span><span>{{ _('We keep your data safe, never sold.') }}</span></p>
```

Comment:
Shouldn't the space be between the `</span> <span>` otherwise all localizers will need to include the space too?

Reply:
You're right the space should be between the `<span></span>`. I'd be careful taking the space out of the `Your privacy comes first.` string though, as even a white space changes can cause translations to break.